### ISSUE TITLE
Error when creating new collection

### ISSUE THREAD

**Issue by gjost** [densho/ddr-cmdln#52]
When creating a new collection the collection gets written to disk but UI shows an error.

NOTE: this is `1_12_2018_newobjerror` in slack.

**Comment by gjost** [comment]
![collection creation error_additional info](https://user-images.githubusercontent.com/38606/35017307-2540e536-fad1-11e7-8973-89a51eeeede6.PNG)

**Comment by gjost** [comment]
Phillip:
```
densho@ddreditor:/var/www/media/ddr/ddr-testing-40040$ cat .git/config | grep uuid
        uuid = b414aaea-23e8-472b-b1d8-4ffbfec4824c
densho@ddreditor:/var/www/media/ddr/ddr-testing-40040$ git annex version
git-annex version: 6.20170101.1
build flags: Assistant Webapp Pairing Testsuite S3(multipartupload)(storageclasses) WebDAV Inotify DBus DesktopNotify ConcurrentOutput TorrentParser MagicMime Feeds Quvi
key/value backends: SHA256E SHA256 SHA512E SHA512 SHA224E SHA224 SHA384E SHA384 SHA3_256E SHA3_256 SHA3_512E SHA3_512 SHA3_224E SHA3_224 SHA3_384E SHA3_384 SKEIN256E SKEIN256 SKEIN512E SKEIN512 SHA1E SHA1 MD5E MD5 WORM URL
remote types: git gcrypt p2p S3 bup directory rsync web bittorrent webdav tahoe glacier ddar hook external
local repository version: 5
supported repository versions: 3 5 6
upgrade supported from repository versions: 0 1 2 3 4 5
operating system: linux x86_64
densho@ddreditor:/var/www/media/ddr/ddr-testing-40040$ git annex info
git-annex: .git/annex/misctmp/mergedrefs.0: createDirectory: permission denied (Permission denied)
failed
git-annex: info: 1 failed
densho@ddreditor:/var/www/media/ddr/ddr-testing-40040$ sudo git annex info
[sudo] password for densho:
repository mode: indirect
trusted repositories: 0
semitrusted repositories: 3
        00000000-0000-0000-0000-000000000001 -- web
        00000000-0000-0000-0000-000000000002 -- bittorrent
        b414aaea-23e8-472b-b1d8-4ffbfec4824c -- ddr@ddreditor:/media/qnfs/kinkura/gold/ddr-testing-40040 [here]
untrusted repositories: 0
transfers in progress: none
available local disk space: 18.19 terabytes (+1 megabyte reserved)
local annex keys: 0
local annex size: 0 bytes
annexed files in working tree: 0
size of annexed files in working tree: 0 bytes
bloom filter size: 32 mebibytes (0% full)
backend usage:
densho@ddreditor:/var/www/media/ddr/ddr-testing-40040$
```

**Comment by gjost** [comment]
The specific problem is that `dvcs.annex_status` is returning `None` and `dvcs._annex_parse_description` is running `annex_status.iterkeys()` on `None`.

Why is `dvcs.annex_status` returning `None`?

**Comment by pkikawa** [comment]
is there anything i can do to check?

**Comment by gjost** [comment]
Might be problem with `git-annex` version.  `git annex version` reports version as `6.20170101.1` but `DDR.dvcs.annex_status` only checks for versions 3 or 5.

**Comment by gjost** [comment]
Should be fixed as of commit 98dc3a8.

Converted the git-annex major version (e.g. 3, 5, 6) into an int.  The if/elif allows for a change in git-annex's UI for getting version info, so choices are now v3 or v5 or above.
